public/main.js

```diff
@@ -37,29 +36,32 @@ function launch(callback) {
 function checkIfUngitIsRunning(callback) {
   // Fastest way to find out if a port is used or not/i.e. if ungit is running
   var net = require('net');
-  var server = net.createServer(function(c) { });
-  server.listen(config.port, function(err) {
-    server.close(function() {
-      callback(null, false);
-    });
-  });
+  var server = net.createServer();
   server.on('error', function (e) {
     if (e.code == 'EADDRINUSE') {
-      callback(null, true);
+      callback(true);
     }
   });
+  server.listen(config.port, config.ungitBindIp, function() {
```

Comment:
This is now the same as here: https://github.com/FredrikNoren/ungit/blob/bdde44bde09221b392842c703b8458c2687eb5d5/source/server.js#L362